.github/workflows/acceptance-tests.yml

```diff
@@ -237,7 +237,6 @@ jobs:
           MONGODB_ATLAS_ORG_ID: ${{ vars.MONGODB_ATLAS_ORG_ID_CLOUD_DEV }}
           MONGODB_ATLAS_BASE_URL: ${{ vars.MONGODB_ATLAS_BASE_URL }}
           MONGODB_ATLAS_ENABLE_BETA: ${{ vars.MONGODB_ATLAS_ENABLE_BETA }}
-          MONGODB_ATLAS_PROJECT_ID: ${{ vars.MONGODB_ATLAS_ACCTEST_STREAMS_PROJECT_ID_CLOUD_DEV }}
```

Comment:
both search_index and stream_connection will not depend on having a fixed project create in our dev environment.